WPFHospitalEditor/Building.xaml.cs

```diff
@@ -19,11 +23,14 @@ namespace WPFHospitalEditor
     public partial class Building : Window
     {
         AllMapObjects allMapObjects = new AllMapObjects();
-        Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
-        HospitalMap hospitalMap = new HospitalMap();
+        public Dictionary<int, Floor> buildingFloors = new Dictionary<int, Floor>();
+        public HospitalMap hospitalMap = new HospitalMap();
+        public int buildingId;
```

Comment:
Why all fields are public and why building needs to have hospitalMap

Reply:
Fields are no longer public, and CanvasService class is created and hospitalMap is no longer needed in Building class.